cmd/snap/cmd_recovery.go

```diff
@@ -53,11 +61,29 @@ func notesForSystem(sys *client.System) string {
 	return "-"
 }
 
+func (x *cmdRecovery) showRecoveryKeys() error {
+	if release.OnClassic {
+		return errors.New(`command "show-recovery-keys" is not available on classic systems`)
+	}
+	var srk *client.SystemRecoveryKeysResponse
+	err := x.client.SystemRecoveryKeys(&srk)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(Stdout, "recovery: %s\n", srk.RecoveryKey)
```

Comment:
do we need to consider escaping or quoting the recovery keys at all?

Reply:
It is put in a ascii friendly format by secboots RecoveryKey.String() helper, something like: 61665-00531-54469-09783-47273-19035-40077-28287

Reply:
if it's missing does the function above error or do we get something like "" ? just wondering because if it for whatever reasons outputs as `-` then this won't be valid yaml and needs to be escaped. I guess I assume that if we don't have the keys then it just errors out earlier? /me will go look at the code

Reply:
seems like it will error out sooner, if the file is empty but exists, as we always verify the size